src/api/index.ts

```diff
@@ -99,3 +101,5 @@ export const tokenListApi: TokenList = createTokenListApi()
 export const erc20Api: Erc20Api = createErc20Api(web3)
 export const depositApi: DepositApi = createDepositApi(erc20Api, web3)
 export const exchangeApi: ExchangeApi = createExchangeApi(erc20Api, web3)
+
+setupAutoconnect(walletApi)
```

Comment:
I feel like this should be in the app or another config place rather than here, although not sure where.